tests/slack_sdk/web/classes/test_blocks.py

```diff
@@ -410,16 +410,16 @@ def test_json(self):
     def test_element_parsing(self):
         elements = [
             ButtonElement(text="Click me", action_id="reg_button", value="1"),
-            StaticSelectElement(options=[Option(value='SelectOption')]),
-            ImageElement(image_url='url', alt_text='alt-text'),
-            OverflowMenuElement(options=[Option(value='MenuOption1'), Option(value='MenuOption2')]),
+            StaticSelectElement(options=[Option(value="SelectOption")]),
```

Comment:
It seems #904 did not apply black (code formatter). Currently, our CI builds do not detect it.

Reply:
I've pushed this change to main branch